recipe/meta.yaml

```diff
@@ -13,39 +13,44 @@ source:
       - 0001-changed-perl-script-to-use-env.patch
 
 build:
-  number: 2
+  number: 3
   run_exports:
     # https://abi-laboratory.pro/?view=timeline&l=gdk-pixbuf
     - {{ pin_subpackage('gdk-pixbuf', max_pin='x') }}
 
 requirements:
   build:
-    - pkg-config
     - {{ compiler('c') }}
-    - meson ==0.56.2         # [(osx and arm64)]
-    - meson >=0.55.3         # [not (osx and arm64)]
-    - ninja-base
-    # is disabled as this package contains wrong path configs
-    - gobject-introspection 1.* # [not win]
-    - perl                   # [win]
-    - python
     - patch                  # [not win]
     - m2-patch               # [win]
+    - gawk  # [not win]
+    - m2-gawk  # [win]
+    - sed   # [not win]
+    - m2-sed  # [win]
```

Comment:
Do we really need `m2-sed` and `m2-gawk`, and `gawk`?  I don't see any usage of it. Or is it for internal building steps?

Reply:
yes, see the bld.bat file.  and sed & gawk are required on unix by some build-tools